t/redirects.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+describe 'redirecting links' do
+  around { |test| VCR.use_cassette('liff', &test) }
+  subject { WikiData.ids_from_pages('en', ['The Meaning of Liff', 'The Deeper Meaning of Liff']) }
```

Comment:
This is a useful test, but I worry that it's a little too lightweight — I can think of all sorts of implementations that could make _this_ code pass, but fail in other scenarios.

I'd suggest leaving this one as is, but also adding a more robust version, that looks up quite a few other pages including at least one that has no redirects in either direction, one where you have a redirect to a page that you haven't specified in your list, and a combination where at least 2 or more different pages all redirect to the same page.